src/main/java/io/quarkus/bot/MarkClosedPullRequestInvalid.java

```diff
@@ -23,6 +25,18 @@ void markClosedPullRequestInvalid(@PullRequest.Closed GHEventPayload.PullRequest
         GHPullRequest pullRequest = pullRequestPayload.getPullRequest();
 
         if (pullRequest.isMerged()) {
+            if (!pullRequest.getLabels().isEmpty()) {
+                if (!quarkusBotConfig.isDryRun()) {
```

Comment:
This test should be around line 33 and you should log the action only if it does something so the LOG.info() line will be inside the if of line 32.